VotingApplication/VotingApplication.Web/Scripts/Controllers/PollHeadingController.js

```diff
@@ -1,17 +1,20 @@
 ﻿(function () {
-    angular.module('GVA.Voting').controller('PollHeadingController', ['$scope', 'PollService', 'ExpiryStringService', function ($scope, PollService, ExpiryStringService) {
-        $scope.pollExpiry = undefined;
+    angular.module('GVA.Voting').controller('PollHeadingController', ['$scope', '$routeParams', 'PollService', 'ExpiryStringService',
```

Comment:
This is rather a long line. Would be a bit easier on the eyes if .module and .controller were each on their own line.